Xamarin.Forms.Platform.iOS/CollectionView/ItemsViewController.cs

```diff
@@ -21,14 +21,23 @@ internal class CollectionViewController : UICollectionViewController
 		UIView _emptyUIView;
 		VisualElement _emptyViewFormsElement;
 
-		public CollectionViewController(ItemsView itemsView, ItemsViewLayout layout) : base(layout)
+		protected UICollectionViewDelegator Delegator { get; set; }
+
+		public ItemsViewController(ItemsView itemsView, ItemsViewLayout layout) : base(layout)
 		{
 			_itemsView = itemsView;
 			_itemsSource = ItemsSourceFactory.Create(_itemsView.ItemsSource, CollectionView);
 			_layout = layout;
 
 			_layout.GetPrototype = GetPrototype;
 			_layout.UniformSize = false; // todo hartez Link this to ItemsView.ItemSizingStrategy hint
+
+			Delegator = new UICollectionViewDelegator
+			{
+				ItemsViewLayout = _layout
+			};
```

Comment:
Just mentioning here - if this is required, maybe make it a constructor arg?

Reply:
Done.